### ISSUE TITLE
revise data entry form

### ISSUE THREAD

**Issue by shapiromatron** [shapiromatron/bmds-server#16]
Update data entry form using the attached mockup:

![Untitled Diagram (1)](https://user-images.githubusercontent.com/999952/83091268-d04d6f00-a068-11ea-878a-57c4ad62c7b3.png)

- remove the modal for dataset selection type
- the "import datasets" does not yet do anything
- the plot size can just be a placeholder for now
- when creating a new dataset, by default create 5 rows
- use `fa-trash` from font awesome instead of 'x'
- the "save dataset" button which was previously present is no longer needed

On the data store, try to exactly mirror the structure of how the `datasets` object is returned from the server. Then it will be easy to pass the data back and forth

The components should roughly be (this is rough; feel free to change)

```html
<DatasetFormContainer>
    <DatasetSelector/>
    <DatasetForm/>
    <DatasetScatterplot/>
<DatasetFormContainer/>
```

There will also be a `selectedDatasetIndex`, and whenever a new dataset is created or a different dataset is selected, this will update, and will change what is displayed on the `DatasetForm` and `DatasetScatterplot` visual components.

The layout above is rough and minor changes are fine, though keeping the three column layout is preferred if possible.

**Comment by shapiromatron** [comment]
@bishwobhandari updates for revising the dataset entry page, after #15 is merged in

**Comment by bishwobhandari** [comment]
successfully merged into master
